README.md

```diff
@@ -2,61 +2,80 @@ Insights Tag Service
 ===========================================
 
 Dev Setup
+
 --------------------
+
 1. Install dependencies
-```
+
```

Comment:
Not sure if it's just me, but the formatting kind of seems off

Reply:
Which part? I added the newline because it's required by markup lint.

Reply:
The whole README :sweat_smile: 
![image](https://user-images.githubusercontent.com/17099954/66924321-619b3400-eff8-11e9-8383-052f4207eaeb.png)

Reply:
Ah, thanks for the screenshot. It looks weird because the section headers aren't being formatted the same way now. I'll throw in a fix

Reply:
Thanks! Looks good!